cli/QuickstepCli.cpp

```diff
@@ -280,7 +279,8 @@ int main(int argc, char* argv[]) {
   Foreman foreman(&bus,
                   query_processor->getDefaultDatabase(),
                   query_processor->getStorageManager(),
-                  num_numa_nodes_covered);
+                  -1, /* CPU id to bind foreman. -1 is unbound. */
```

Comment:
@hbdeshmukh Is there a need to bind the foreman to a node? I set it to unbound because it seems like a good default.